actionview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add `config.active_model.i18n_full_message` in order to control whether
```

Comment:
Should be in `activemodel/CHANGELOG.md`.

Reply:
Yes. It is already there.